test/controllers/content_items_controller_test.rb

```diff
@@ -119,14 +119,13 @@ class ContentItemsControllerTest < ActionController::TestCase
 
     content_store_has_item(content_item['base_path'], content_item)
 
-    with_variant EducationNavigation: "A" do
+    with_variant EducationNavigation: "A", assert_meta_tag: false do
```

Comment:
The gem is memoising `analytics_meta_tags`, which means that when we use `with_variant` more than once in the same test case, we keep the first meta tag. This is a bug, which I am fixing here: https://github.com/alphagov/govuk_ab_testing/pull/24

When that's merged, I will bump the version of the gem here and remove that `assert_meta_tag: false`.